packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -90,10 +99,47 @@ export default class QuanticResultLink extends LightningElement {
     );
   };
 
+  checkResultType() {
+    if (this.result?.raw?.sfid !== undefined) {
+      this.isSalesforceLink = true;
+    }
+  }
+
+  handleOpenTabSalesforceLink(event) {
+    event.stopPropagation();
+
+    const targetPageRef = {
+      type: 'standard__recordPage',
+      attributes: {
+        recordId: this.getRecordIdAttribute(),
+        objectApiName: this.getObjectApiNameAttribute(),
+        actionName: 'view',
+      },
+    };
+    this[NavigationMixin.Navigate](targetPageRef);
+  }
+
+  getRecordIdAttribute() {
```

Comment:
This method cannot afford to return `''` as it is now as you are taking the return value directly.

Right now is we don't have the `sfid` nor the `sfkbid|sfkavid` this will return an empty string and you will try to open a `standard__recordPage` with an empty ID.

I suggest that you get the return of this method first, and if you see this returns undefined (don't return empty string, it is less clear that the method didn't find what it was looking for than if it returns undefined), you instead treat the result link as a regular result link and simply open the clickUri as if it was a normal result.

Reply:
ok I will change the getter to this:

`return sfid !== undefined && sfid !== null;`

so that by the time it gets the recordId the sfid is for sure not null or undefined. If it is `isSalesforceLink` is false and the link will simply default to an a tag with `href={result.clickUri}`